parquet-column/src/test/java/org/apache/parquet/column/values/Utils.java

```diff
@@ -61,7 +62,7 @@ public static void writeData(ValuesWriter writer, String[] strings)
   public static Binary[] readData(ValuesReader reader, byte[] data, int offset, int length)
       throws IOException {
     Binary[] bins = new Binary[length];
-    reader.initFromPage(length, data, 0);
+    reader.initFromPage(length, ByteBuffer.wrap(data), 0);
```

Comment:
Deprecated the old one and removed all current references to it in the codebase.